BattleTank/Game1.cs

```diff
@@ -638,6 +636,7 @@ protected override void Update(GameTime gameTime)
                         ButtonSettings = this.Content.Load<Texture2D>("Graphics//settings1");
                         if (state.LeftButton == ButtonState.Pressed)
                         {
+                            AvailableGamepads = GamePads.GetAllAvailableGamepads();
```

Comment:
W linii 170 jest napisane to samo, ta powtórka jest konieczna do tego, aby to działało?

Reply:
Tak, wydaje mi się że ok jest sprawdzenie dostepnych padów przed konfiguracją sterowania